ui/src/fragments/Tabs.svelte

```diff
@@ -5,11 +5,13 @@
 <div class="flex gap-4 border-b">
   {#each tabs as tab}
     <button
-      class="cursor-pointer text-lg hover:text-black font-saucebold hover:duration-500 dark:text-white dark:hover:text-offwhite"
-      class:border-b-black={activeTab === tab}
+      class="cursor-pointer text-lg hover:text-black font-saucebold hover:duration-500 dark:border-b-[#ffffff] dark:hover:text-white"
+      class:border-b-[#000000]={activeTab === tab}
```

Comment:
these don't work properly without inline hex codes. do we need to extend custom colors to borders in tailwind.config.css?

Reply:
Yes, there is already a section in the tailwind config file for border colours.